compiler/dredd-rule-lib/rule-lib.sh

```diff
@@ -217,4 +217,21 @@ op_version()
   echo ${ACTUAL}
 }
 
+tensor_dtype()
+{
+  argc_check $# 1
+  file_path_check ${COMPILED_FILE}
+  file_path_check ${INSPECT_PROG_PATH}
+
+  set -o pipefail
+
+  ACTUAL=`init_error_log ; \
+          ${INSPECT_PROG_PATH} --tensor_dtype ${COMPILED_FILE} | \
+          awk -v opname="$1" '{ if ($1 == opname) print $2}'`
```

Comment:
I am not sure but maybe... `tensorname` instead of `opname` ?

Reply:
IMHO it seems that `tensor_name` instead of `opname`? (since ifm or weight is not opname)

_from README.md_ in draft,
```
$ circle-inspect --tensor_dtype quantized_conv2d.circle

Result is...

ifm UINT8
weights UINT8
bias INT32
ofm UINT8
```

Reply:
tensor_name would be appropriate :)